src/renderer/components/icon/icon.tsx

```diff
@@ -15,6 +15,7 @@ export interface IconProps extends React.HTMLAttributes<any>, TooltipDecoratorPr
   href?: string;              // render icon as hyperlink
   size?: string | number;     // icon-size
   small?: boolean;            // pre-defined icon-size
+  xs?: boolean;            // pre-defined icon-size
```

Comment:
Maybe better `smallest` or `smaller` name (to be more aligned with other names)?